### ISSUE TITLE
Fee informations

### ISSUE THREAD

**Issue by sabau** [cosmos/voyager#1721]
UI Version: 0.X.X

Description:
For a user is a bit hard to understand that he's paying Gas for transactions and to retrive this information later. It would be nice to have in the Voting, Depsit and Transaction actions and show it in a confirmation box: The fee of this operation is X

**Comment by fedekunze** [comment]
I think the user should eventually define the gas and fees (amount + coin denomination) by herself  and also provide the option in the settings to define a default gas and fee

**Comment by fedekunze** [comment]
cc: @alessio what's the cmd/parameter to estimate the gas cost ?

**Comment by jbibla** [comment]
absolutely - we need to do a better job of communicating gas and fees!

**Comment by fedekunze** [comment]
After the discussion in the sdk channel, I think gas is not relevant to our end users. Clients (voyager) could run a simulation previously and check the estimated gas used and set that value on the gas field. 

Regarding fees, imo we should eventually define an input field within the txs modals (Deposit, Vote, Delegate, etc) and have a custom value set as default on the preferences/settings page that's pre-filled on the modals

**Comment by faboweb** [comment]
Proposal:
Don't show the concept of gas to the user. The user only cares about fees he needs to pay (which include gas). Voyager shouldn't care about gas either. Stargate should handle fee calculation as it can reuse calculated minimum fees across multiple requests. A client could request stored minimum fees from Stargate similar to the current "simulate" flag.

**Comment by fedekunze** [comment]
Scope:

- [ ] Input transaction fees on the tx modals
- [ ] Run the tx simulation and propose a minimum fee to the input

**Comment by jbibla** [comment]
1. use tx simulation to get gas price
2. convert gas price into atoms
3. display this price to users
4. if gas price is too low and transaction is dropped, show button for increasing gas price by x% so user can try again (in action modal)

- don't show "gas" to users - just show atoms as tx fee

**Comment by faboweb** [comment]
> 4\. if gas price is too low and transaction is dropped, show button for increasing gas price by x% so user can try again (in action modal)

I am not sure we can identify this. Transactions will go to the mempool ready to be claimed by a block proposer and will only after several blocks be dropped.

**Comment by fedekunze** [comment]
moved to #2238
